hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/metadata/SCMDBDefinition.java

```diff
@@ -105,6 +114,6 @@ public String getLocationConfigKey() {
   @Override
   public DBColumnFamilyDefinition[] getColumnFamilies() {
     return new DBColumnFamilyDefinition[] {DELETED_BLOCKS, VALID_CERTS,
-        REVOKED_CERTS, PIPELINES, CONTAINERS, TRANSACTIONINFO};
+        VALID_SCM_CERTS, REVOKED_CERTS, PIPELINES, CONTAINERS, TRANSACTIONINFO};
```

Comment:
we might also need a REVOKED_SCM_CERTS with the CRL work given we are putting cert of different roles into different table with this change.

Reply:
Right now added scmCert to validCertsTable, so that revoke will remove from both for Scmcerts.
 I think once we have a separate table for each role, we can add that.

Reply:
Yes, might be for list for role SCM for certtype RevokedCerts we might need this. Figured out during implementation of listCertificates. As for security work this is not required, will open a new Jira to fix this later.